pkg/pgp/pgp.go

```diff
@@ -138,9 +78,40 @@ func DecodePgpKey(pgpKey, path string) (*openpgp.Entity, error) {
 
 // DecodeAndArmorBase64Entity decodes a base64 encoded entity and armors it.
 func DecodeAndArmorBase64Entity(encodedEntity string, armorType string) (string, error) {
```

Comment:
Will this work with already armored keys?

Reply:
I think the code earlier never let us add an armored key in user.yml. Most of the keys are base64 encoded keys. (without checksum)